Backend/app/rider-platform/rider-app/Main/spec/Storage/ticket.yaml

```diff
@@ -276,10 +328,14 @@ TicketBookingService:
     amount: Price
     status: ServiceStatus
     verificationCount: Int
+    visitDate: Maybe Day
     btype: BusinessHourType
+    bHourId: Maybe (Id BusinessHour)
```

Comment:
Why is this added?

Reply:
we need businessHourId to handle the lock in user side, such that merchant can't cancel that service category of that businessHour at that visitDate